docs/store/README.md

```diff
@@ -96,7 +96,7 @@ export class MyAppComponent {
 	counter: Observable<number>;
 
 	constructor(private store: Store<AppState>) {
-		this.counter = store.select<number>('counter');
+		this.counter = store.select<number>((state: AppState) => state.counter);
```

Comment:
You were right, I guess it is easier for new comers to select state using the `string` selector instead of the callback.